packages/core/src/provider/initConfigs.ts

```diff
@@ -33,16 +34,33 @@ async function modifyRsbuildConfig(context: InternalContext) {
   logger.debug('modify Rsbuild config done');
 }
 
+async function modifyEnvironmentConfig(
+  context: InternalContext,
+  config: MergedEnvironmentConfig,
+  name: string,
+) {
+  logger.debug(`modify Rsbuild environment(${name}) config`);
+  const [modified] = await context.hooks.modifyEnvironmentConfig.call(config, {
+    name,
+    // @ts-expect-error
```

Comment:
Add the reason for @ts-expect-error
```suggestion
    // @ts-expect-error some reason
```